spec/views/envents/index.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe "events/index.html.erb" do
+  it "displays all the events" do
+    first_event = create(:event)
+    second_event = create(:event, title: "event two")
+    assign(:events, [first_event, second_event])
+    render
+
+    expect(rendered).to match /#{first_event.title}/
+    expect(rendered).to match /#{second_event.title}/
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.